spec/bowlingGameSpec.js

```diff
@@ -0,0 +1,50 @@
+describe('BowlingGame', function() {
+
+  beforeEach(function () {
+    game = new BowlingGame();
+  });
+
+  it('should begin with no score', function () {
+      expect(game.rolls).toEqual([]);
```

Comment:
An alternative could be to test the length `expect(game.rolls.length).toEqual(0)`